src/main/java/edu/maebe/sql2omodel/Sql2oModel.java

```diff
@@ -189,6 +190,54 @@ public boolean existNeed(UUID need) {
         }
     }
 
+    //userSettings
+    @Override
+    public UUID createUserSettings(String user, Boolean readFeedbackImmediately, String emailAddress, String phoneNumber,
+                                   int numberOfChildren, String cellProvider, Date lastUpdate){
+        try (Connection conn = sql2o.beginTransaction()) {
+            UUID userSettingsId = uuidGenerator.generate();
+            conn.createQuery("insert into userSettings (id, user, readFeedbackImmediately, emailAddress," +
+                    " phoneNumber, numberOfChildren, cellProvider, lastUpdate) values (:id, :user, :readFeedbackImmediately," +
+                    " :emailAddress, :phoneNumber, :numberOfChildren, :cellProvider, :lastUpdate)")
+                    .addParameter("id", userSettingsId)
+                    .addParameter("user", user)
+                    .addParameter("readFeedbackImmediately", readFeedbackImmediately)
+                    .addParameter("emailAddress", emailAddress)
+                    .addParameter("phoneNumber", phoneNumber)
+                    .addParameter("numberOfChildren", numberOfChildren)
+                    .addParameter("cellProvider", cellProvider)
+                    .addParameter("lastUpdate", new Date())
+                    .executeUpdate();
+            conn.commit();
+            return  userSettingsId;
+        }
+
+    }
+
+    @Override
+    public UserSettings getUserSettings(String user){
+        try (Connection conn = sql2o.beginTransaction()) {
+            List<UserSettings> settings = conn.createQuery("select * from userSettings where alex = :user")
+                    .addParameter("user", user)
```

Comment:
you will want to add the column mappings to show how you transfer from the column names to the class property names (because different casing and such)